### PR TITLE
DG update

### LINKED ISSUE #54: Update the developer guide to include features
- [x] Design / Architecture
- [x] Implementation
- [x] Product scope (Target user profile and Value proposition)
- [x] User Stories
- [x] Non-Functional Requirements
- [x] Glossary
- [x] Instructions for manual testing

## Updates

- [x] See [prof's suggestions](https://github.com/nus-cs2113-AY2021S1/forum/issues/112)

### PR DESCRIPTION
**Pull request by yuxinwu99**
Add list stops and fixed some format issues

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -160,10 +162,23 @@ The following sequence diagram explains the above steps when the user searches f
 
 ![Overview](DG_Diagrams/RouteMapCommandSeq.png)
 
+###3.3. List All stops (/liststops Feature)
+`/liststops` is the command which prints all bus stops declared in the BusStops enum.
+
+The `ListStopsCommand#executeCommand()` method of ListStopsCommand Class executes the command in the following steps:
```
Comment (Johnson-Yee) [inline docs/DeveloperGuide.md:169]:
```suggestion
The `ListStopsCommand#executeCommand()` method of ListStopsCommand Class executes the command in the following step:
```

**Review by Johnson-Yee — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wamikamalik) [reply docs/DeveloperGuide.md:169]: I think this section is still incomplete?

Comment (yuxinwu99) [reply docs/DeveloperGuide.md:169]: yeah

Comment (yuxinwu99) [reply docs/DeveloperGuide.md:169]: i forgot it updates automatically hahaha

**Pull request merged**